.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Run lint
+        run: yarn lint
+      - name: Build Components
+        run: yarn build
+
+      - name: Install dependencies
+        working-directory: wallets-testing
+        run: yarn install --immutable
+      - name: Run walllets tests
+        run: yarn test
+        working-directory: wallets-testing
+        env:
+          RPC_URL: ${{ secrets.RPC_URL }}
+#      - name: Publish to NPM
+#        env:
+#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+#        run: yarn multi-semantic-release --deps.bump=override --deps.release=patch --sequential-init
```

Comment:
why commented?

Reply:
there's no npm tokens at the moment, will uncomment it a bit late